nishimweelysee/Elysee-CapstoneProject
# FT User Blog Page

#### What does this PR do?
> - This PR made with a branch that display the blogs posted by the Blog owner so the users will be able to see it.
>
#### Description of Task to be completed
> - Image of the Blog
> - Content of the Blog
>
#### How should this be manually tested?
> - Open a website then go to blog in navigation menu
>- Then you will see a blog displayed as grids.
#### Any background context you want to provide?
> that is a more blog page and there will be a single blog page.
#### What are the relevant Trello story?
> Ft(User Blog Page) https://trello.com/c/HIVCWsaj Use this link to see my trello board
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92329469-e73c2e00-f067-11ea-97b5-1849063c2c69.png)

![image](https://user-images.githubusercontent.com/56052588/92329479-fd49ee80-f067-11ea-84d8-9cd960a6c127.png)

Here is a screenshot for blog page. they will be one on top that will be display all blog as a motion picture.

Thank you!
